pkgs/misc/uboot/default.nix

```diff
@@ -150,12 +152,18 @@ in rec {
     filesToInstall = ["u-boot-sunxi-with-spl.bin"];
   };
 
-  ubootQemuArm = buildUBoot rec {
+  ubootQemuArm_32bit = buildUBoot rec {
     defconfig = "qemu_arm_defconfig";
     targetPlatforms = ["armv7l-linux"];
     filesToInstall = ["u-boot.bin"];
   };
 
+  ubootQemuArm_64bit = buildUBoot rec {
```

Comment:
This could be `ubootQemuAarch64`, in Nixpkgs naming 'Arm' always stands for the 32-bit one.